WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -873,6 +873,21 @@ - (BlogDetailsSection *)generalSectionViewModel
                                                          [weakSelf showActivity];
                                                      }]];
     }
+    
+    if ([self shouldShowBlaze]) {
+        CGSize iconSize = CGSizeMake(BlogDetailGridiconSize, BlogDetailGridiconSize);
+        UIImage *blazeIcon = [[UIImage imageNamed:@"icon-blaze"] resizedImage:iconSize interpolationQuality:kCGInterpolationHigh];
+        BlogDetailsRow *blazeRow = [[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Blaze", @"Noun. Links to a blog's Blaze screen.")
+                                                 accessibilityIdentifier:@"Blaze Row"
+                                                                   image:[blazeIcon imageFlippedForRightToLeftLayoutDirection]
+                                                              imageColor:nil
+                                                           renderingMode:UIImageRenderingModeAlwaysOriginal
+                                                                callback:^{
+                                                                    [weakSelf showBlaze];
+                                                                }];
+        blazeRow.showsSelectionState = NO;
```

Comment:
[Nit] What do you think of extracting this to a helper that creates a Blaze row to avoid code duplication?

Reply:
Done! [10e18e7](https://github.com/wordpress-mobile/WordPress-iOS/pull/20314/commits/10e18e790e80835dcd63bc70ccf0da0b31cb31f9)